packages/kbn-securitysolution-io-ts-list-types/src/request/index.ts

```diff
@@ -23,6 +23,8 @@ export * from './find_exception_list_item_schema';
 export * from './find_list_item_schema';
 export * from './find_list_schema';
 export * from './import_list_item_query_schema';
+export * from './import_exception_list_schema';
```

Comment:
I know we're moving away from this, hoping to do a sweeping PR to remove these from lists plugin.